artifactory/commands/transferfiles/delayedartifactshandler_test.go

```diff
@@ -50,8 +60,31 @@ func TestDelayedArtifactsMng(t *testing.T) {
 	artifactsChannelMng.close()
 	readWaitGroup.Wait()
 	assert.NoError(t, delayedArtifactsErr)
+
+	// add not relevant files to confuse
+	for i := 0; i < 4; i++ {
+
+		assert.NoError(t, ioutil.WriteFile(filepath.Join(delaysDirPath, getDelayMockFileName(delayTestRepoKey, i)), nil, 0644))
+	}
+	assert.NoError(t, ioutil.WriteFile(filepath.Join(delaysDirPath, getDelayMockFileName("wrong-"+testRepoKey, 1)), nil, 0644))
+	assert.NoError(t, ioutil.WriteFile(filepath.Join(delaysDirPath, getDelayMockFileName(testRepoKey+"-wrong", 0)), nil, 0644))
+	assert.NoError(t, ioutil.WriteFile(filepath.Join(delaysDirPath, getDelayMockFileName(testRepoKey+"-0-0", 0)), nil, 0644))
+	assert.NoError(t, ioutil.WriteFile(filepath.Join(delaysDirPath, getDelayMockFileName(testRepoKey+"-1", 0)), nil, 0644))
+	assert.NoError(t, ioutil.WriteFile(filepath.Join(delaysDirPath, getDelayMockFileName("wrong-"+testRepoKey+"-wrong", 0)), nil, 0644))
```

Comment:
This line repeats 6 times:
```go
assert.NoError(t, ioutil.WriteFile(filepath.Join(delaysDirPath, getDelayMockFileName("xxx", 111)), nil, 0644))
```
Can we add a loop or use a dedicated function to reduce code duplication?